src/pybind/mgr/mgr_module.py

```diff
@@ -304,6 +311,20 @@ def __del__(self):
     def log(self):
         return self._logger
 
+    def cluster_log(self, channel, priority, record):
+        """
+        :param channel: The log channel. This can be 'cluster', 'audit', ...
+        :type channel: string
+        :param priority: The log message priority. This can be
+                         CLUSTER_LOG_PRIO_DEBUG, CLUSTER_LOG_PRIO_INFO,
+                         CLUSTER_LOG_PRIO_SEC, CLUSTER_LOG_PRIO_WARN or
+                         CLUSTER_LOG_PRIO_ERROR.
+        :type priority: integer
```

Comment:
`s/integer/int/` to make it work with static type checking.

Reply:
Done